File: testcases/src/main/java/root_paths/RestApp1.java
package root_paths;

import javax.ws.rs.ApplicationPath;
import javax.ws.rs.core.Application;
import java.util.HashSet;
import java.util.Set;

//TODO check with 2 root paths
@ApplicationPath("/root_path1")
public class RestApp1 extends Application {

    @Override
    public Set<Class<?>> getClasses() {
        Set<Class<?>> sets = new HashSet<Class<?>>();
//        sets.add();
        return sets;
    }
}

File: testcases_kt/src/main/kotlin/root_paths/RestApp1.kt
package root_paths

import javax.ws.rs.ApplicationPath
import javax.ws.rs.core.Application

//TODO check with 2 root paths
@ApplicationPath("/root_path1_kt")
class RestApp1 : Application() {
    override fun getClasses(): Set<Class<*>> {
        //        sets.add();
        return HashSet()
    }
}
